docs/content/posts/2024-03-xx-release-1.19.0.md

```diff
@@ -0,0 +1,466 @@
+
+-- apply aggregation on the session windowed table with partition keys
+SELECT window_start, window_end, item, SUM(price) AS total_price
+FROM TABLE(
+    SESSION(TABLE Bid PARTITION BY item, DESCRIPTOR(bidtime), INTERVAL '5' MINUTES))
+GROUP BY item, window_start, window_end;
+```
+* Supports Changelog Inputs For Window TVF Aggregation
+  Window aggregation operators (generated based on Window TVF Function) can now handle changelog
+  streams (e.g., CDC data sources, etc.) without issue. Users are recommended to migrate from legacy
+  window aggregation to the new syntax for more complete feature support.
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/window-tvf/#session)
+
+
+### MiniBatch Optimization for Regular Joins
+
+##### [FLINK-34219](https://issues.apache.org/jira/browse/FLINK-34219)
+
+The record amplification is a pain point when performing cascading joins in Flink, now in Flink 1.19,
+the new mini-batch optimization can be used for regular join to reduce intermediate result in such
+cascading join scenarios.
+
+<div style="text-align: center;">
+<img src="/img/blog/2024-03-xx-release-1.19.0/minibatch_join.png" style="width:90%;margin:15px">
+</div>
+
+**More Information**
+* [minibatch-regular-joins](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/dev/table/tuning/#minibatch-regular-joins).
+* [FLIP-415: Introduce a new join operator to support minibatch](https://cwiki.apache.org/confluence/display/FLINK/FLIP-415%3A+Introduce+a+new+join+operator+to+support+minibatch)
+
+# Key Improvements
+
+## Runtime & Coordination
+
+### Support dynamic source parallelism inference for batch jobs
+
+In Flink 1.19, we have supported dynamic source parallelism inference for batch jobs, which allows
+source connectors to dynamically infer the parallelism based on the actual amount of data to consume.
+This feature is a significant improvement over previous versions, which only assigned a fixed default
+parallelism to source vertices.
+Source connectors need to implement the inference interface to enable dynamic parallelism inference.
+Currently, the FileSource connector has already been developed with this functionality in place.
+Additionally, the configuration `execution.batch.adaptive.auto-parallelism.default-source-parallelism`
+will be used as the upper bound of source parallelism inference. And now it will not default to 1.
+Instead, if it is not set, the upper bound of allowed parallelism set via
+`execution.batch.adaptive.auto-parallelism.max-parallelism` will be used. If that configuration is
+also not set, the default parallelism set via `parallelism.default` or `StreamExecutionEnvironment#setParallelism()`
+will be used instead.
+
+**More Information**
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/deployment/elastic_scaling/#enable-dynamic-parallelism-inference-support-for-sources).
```

Comment:
I noticed that some doc links are referred as `https://nightlies.apache.org/flink/flink-docs-release-1.19/`, while some are referred as `https://nightlies.apache.org/flink/flink-docs-master`, can we make them align together?